plugin/contrib/mesos/pkg/scheduler/plugin_test.go

```diff
@@ -511,6 +515,9 @@ func TestPlugin_LifeCycle(t *testing.T) {
 	mockDriver.AssertNumberOfCalls(t, "LaunchTasks", 1)
 	assert.Equal(1, len(launchTasks_taskInfos))
 
+	// check that ExecutorInfo.data has the static pod data
+	assert.Len(launchTasks_taskInfos[0].Executor.Data, 3)
```

Comment:
That probability is as small as the cost of testing the actual bytes, instead of the length :)